pyriemann/classification.py

```diff
@@ -514,3 +515,142 @@ def predict(self, covtest):
+    For classification, a given new point is attibuted to the class whose centroid is 
+    the nearest according to the chosen metric.
+
+    Parameters
+    ----------
+    metric : string | dict (default: 'riemann')
+        The type of metric used for centroid and distance estimation.
+        see `mean_covariance` for the list of supported metric.
+        the metric could be a dict with two keys, `mean` and `distance` in
+        order to pass different metric for the centroid estimation and the
+        distance estimation. Typical usecase is to pass 'logeuclid' metric for
+        the mean in order to boost the computional speed and 'riemann' for the
+        distance in order to keep the good sensitivity for the classification.
+    L : float, (default: 0)
+        Transfer coefficient. This parameter controls the trade-off between 
+        source and target data.
+    n_jobs : int, (default: 1)
+        The number of jobs to use for the computation. This works by computing
+        each of the class centroid in parallel.
+        If -1 all CPUs are used. If 1 is given, no parallel computing code is
+        used at all, which is useful for debugging. For n_jobs below -1,
+        (n_cpus + 1 + n_jobs) are used. Thus for n_jobs = -2, all CPUs but one
+        are used.
+
+    Attributes
+    ----------
+    covmeans_ : list
+        the class centroids.
+    classes_ : list
+        list of classes.
+
+    See Also
+    --------
+    MDM
+    Kmeans
+    FgMDM
+    KNearestNeighbor
+
+    References
+    ----------
+    [1] E. Kalunga, S. Chevallier and Q. Barthélemy, "Transfer learning for 
+    SSVEP-based BCI using Riemannian similarities between users", in 26th 
+    European Signal Processing Conference (EUSIPCO), pp. 1685-1689. IEEE, 2018.
+
+    [2] S. Khazem, S. Chevallier, Q. Barthélemy, K. Haroun and C. Noûs, 
+    "Minimizing Subject-dependent Calibration for BCI with Riemannian Transfer 
+    Learning", in 10th International IEEE/EMBS Conference on Neural 
+    Engineering (NER), pp. 523-526. IEEE, 2021.
+    """
+
+    def __init__(self,metric='riemann', L=0, n_jobs=1):
+        
+        """Init."""
+        self.metric = metric
+        self.n_jobs = n_jobs
+        self.L = L
+
+        if isinstance(metric, str):
+            self.metric_mean = metric
+            self.metric_dist = metric
+
+        elif isinstance(metric, dict):
+            # check keys
+            for key in ['mean', 'distance']:
+                if key not in metric.keys():
+                    raise KeyError('metric must contain "mean" and "distance"')
+
+            self.metric_mean = metric['mean']
+            self.metric_dist = metric['distance']
+
+        else:
+            raise TypeError('metric must be dict or str')
+
+
+    def fit(self, X, y, X_source, y_source, sample_weight=None):
+        """Fit (estimates) the centroids.
+
+        Parameters
+        ----------
+        X : ndarray, shape (n_trials, n_channels, n_channels)
```

Comment:
n_trials must be replaced by n_matrices:
```suggestion
        X : ndarray, shape (n_matrices, n_channels, n_channels)
```

Reply:
This modification should be applied in all parameter descriptions.